mrp_production_request/models/mrp_production_request.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2017-19 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+import odoo.addons.decimal_precision as dp
+from odoo.exceptions import UserError
+
+
+class MrpProductionRequest(models.Model):
+    _name = "mrp.production.request"
+    _description = "Manufacturing Request"
+    _inherit = "mail.thread"
+    _order = "date_planned_start desc, id desc"
+
+    @api.model
+    def _company_get(self):
+        company_id = self.env['res.company']._company_default_get(self._name)
```

Comment:
Because of https://github.com/OCA/ocb/blob/12.0/odoo/addons/base/models/res_company.py#L166-L167

```suggestion
        company_id = self.env['res.company']._company_default_get()
```